feeder/util/mqtt/client.py

```diff
@@ -84,6 +84,12 @@ class FeederClient(MQTTClient):
     api_regex = re.compile(r"^krs\.api\.gts\.(?P<gateway_id>.*)$")
     telemetry_regex = re.compile(r"^krs\.tel\.gts\.(?P<gateway_id>.*)$")
 
+    def __init__(self, host="localhost", port=1883, scheme="mqtt", username=local_username, password=local_password):
```

Comment:
Any specific reason you are passing this in on initialization, versus just passing everything here to the `start` function?

Reply:
Just old "convention" of "these shouldn't change, so put them in the class/object init function" instead of putting them in the start up method.